camelot/model/memento.py

```diff
@@ -106,7 +109,3 @@ class Admin( EntityAdmin ):
                                         'delegate':delegates.One2ManyDelegate,
                                         'python_type':list}
                             }
-        
-    Admin = not_editable_admin( Admin )
```

Comment:
bvb in het geval van de memento, is deep en no actions etc nt van toepassing, dus die 'eenvoudige' check op niveau van de class zou voldoende moeten zijn